docs/guide/front-end-assets.md

```diff
@@ -68,18 +73,55 @@ The output will **sing** before it **dances**. If the modules are instantiated i
+:::
+
+## Supporting Internet Explorer 11
+
+Internet Explorer 11 is [going away soon](https://blogs.windows.com/windowsexperience/2021/05/19/the-future-of-internet-explorer-on-windows-10-is-in-microsoft-edge/#:~:text=With%20Microsoft%20Edge%20capable%20of,certain%20versions%20of%20Windows%2010.), but for now at least some projects still require support for it.
 
-**Here is how development works using our boilerplate project**:
+By default, Apostrophe does *not* guarantee that code written in modern JavaScript in `ui/src/index.js` files will run in Internet Explorer 11. However, you can easily turn on support by setting the `es5: true` option of the `@apostrophecms/asset` module:
+
+```js
+// modules/@apostrophecms/asset/index.js
+module.exports = {
+  options: {
+    es5: true
+  }
+};
+```
 
--  The `dev` npm script in `package.json` runs a project-level webpack build, which compiles `src/index.js` according to the rules in `webpack.config.js`.
--  `src/index.scss` (written in [Sass](https://sass-lang.com/)) is imported by `src/index.js`, allowing the browser to load one file for both CSS and JS.
--  At the end of the build, the bundled assets are written to `modules/asset/ui/public/sites.js`.
-- Any `.js` files in the `ui/public` folder of any module are automatically included in the frontend build.
-- Whenever code changes are made, `nodemon` automatically restarts this cycle and refresh the browser after a successful restart.
+This configures Apostrophe to compile two JavaScript bundles: one for Internet Explorer 11 users, and one for everyone else. The Internet Explorer 11 bundle contains code compiled with `babel` in order to add support for missing features, as well as "polyfills" for missing functions.
 
-None of those specifics are required for Apostrophe. You can follow whatever organizing pattern works best for you or your organization.
+That makes the bundle larger of course, so Apostrophe serves a separate, small and fast bundle to non-IE11 users so there is **no performance penalty** for them.
+
+::: note Internet Explorer 11 Limitations
+Apostrophe uses babel's `@babel/preset-env` module to provide as much support as possible for JavaScript language features, including promises, `async/await` and more. However, Apostrophe does not provide polyfills for all browser API features, and some, such as `Observable`, cannot be implemented for Internet Explorer 11. Apostrophe also does not attempt to fix the limitations of CSS in Internet Explorer 11. And of course, the most obvious limitation is that **Apostrophe's editing interface is not available in Internet Explorer 11.** You should test your sites thoroughly in all browsers you intend to support.
 :::
+
+## Using your own Webpack configuration
```

Comment:
Quibble: It's "webpack," lowercase because grammar no longer means anything.

Reply:
But... I don't love us making it sound like it has to be webpack. This is all about people owning their own lives, so " your own build process" would be better. Other mentions in this section as well.

Reply:
Agree that if webpack is not a requirement, then we shouldn't communicate like it is one here. The idea is for this to be agnostic about the front-end build tool right?

Reply:
I think people who don't use webpack, but do use a build process, know exactly what they are doing. Right now the people who most need our help are probably in webpack. I could go with "your own build process, such as webpack". I don't want us to lose the SEO value of saying the word they are almost certainly searching for.

Reply:
I don't mind "such as webpack." My two cents of snarkiness is that the people who most need the help are those who feel they need to use webpack because that's all everyone talks about.